.circleci/src/jobs/promote_web.yml

```diff
@@ -0,0 +1,28 @@
+docker:
+  - image: cimg/node:14.19.0
```

Comment:
is there a reason why we don't use LTS Version? 16.15.1

Reply:
Just not updated yet, feel free to create an issue to update all docker images in the CI